registries/_format/base64url.md

```diff
@@ -0,0 +1,136 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-5)
```

Comment:
I'm a bit confused at the combination of "binary array", which sounds like the datatype you are targeting, along with "url-safe", which refers to the encoded text.  I think what you want here is:

> Binary data encoded as a url-safe string as defined...

I would not mention the target data type at all, as the appropriate data type will vary among languages.  For example, I'm not sure what I'd do with "binary array" in Python.  In my view, the OAS should trust implementers to pick the right way to represent binary data, and avoid expressing a preference.

Reply:
```suggestion
description: Binary data encoded as a url-safe string as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-5)
```